docs/deployed-contracts/sepolia.md

```diff
@@ -100,6 +100,21 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
   - L2LidoGateway
   - L2WstETHToken
 
+### Mode
+
+##### Ethereum part
+
+- L1ERC20TokenBridge: [`0xD847780c8808fc99727001C43E6C17A198b39729`](https://sepolia.etherscan.io/address/0xD847780c8808fc99727001C43E6C17A198b39729) (proxy)
+- L1ERC20TokenBridge: [`0x514Aa1d41d840af7572509D75354A8612594eB30`](https://sepolia.etherscan.io/address/0x514Aa1d41d840af7572509D75354A8612594eB30) (impl)
```

Comment:
should be https://sepolia.etherscan.io/address/0x8c1E68A74E71594925c7D7a78Ef43657aec4d599